CMakeLists.txt

```diff
@@ -1147,81 +1159,100 @@ IF(PCRE2_HEAP_MATCH_RECURSE)
 ENDIF(PCRE2_HEAP_MATCH_RECURSE)
 
 IF(PCRE2_SHOW_REPORT)
-  STRING(TOUPPER "${CMAKE_BUILD_TYPE}" buildtype)
-  IF (CMAKE_C_FLAGS)
-    SET(cfsp " ")
-  ENDIF(CMAKE_C_FLAGS)
   MESSAGE(STATUS "")
   MESSAGE(STATUS "")
   MESSAGE(STATUS "PCRE2-${PCRE2_MAJOR}.${PCRE2_MINOR} configuration summary:")
   MESSAGE(STATUS "")
-  MESSAGE(STATUS "  Install prefix .................. : ${CMAKE_INSTALL_PREFIX}")
-  MESSAGE(STATUS "  C compiler ...................... : ${CMAKE_C_COMPILER}")
-  MESSAGE(STATUS "  C compiler flags ................ : ${CMAKE_C_FLAGS}${cfsp}${CMAKE_C_FLAGS_${buildtype}}")
+  MESSAGE(STATUS "  Install prefix .................... : ${CMAKE_INSTALL_PREFIX}")
+  MESSAGE(STATUS "  C compiler ........................ : ${CMAKE_C_COMPILER}")
+  if (CMAKE_CONFIGURATION_TYPES)
+    FOREACH (config IN LISTS CMAKE_CONFIGURATION_TYPES)
```

Comment:
nitpick: can we use lowercase foreach?

Reply:
OK, done.

But someone really needs to just run an autoformatter over the file, it's totally inconsistent at the moment. I can do that in a follow-up.

Reply:
the inconsistency is by design, see #115

Reply:
Well 23 instances of "if" lowercase and 124 instances of "IF" can't be by design. The formatting should at least be consistent. It's not important though, given it has absolutely no functional impact.

Making it more idiomatic CMake would be nice in future, there are some good ideas in that thread.

Reply:
Sorry, I meant "by design" ironically, since that file has been hacked without any for way too long IMHO.

Indeed, I have to admit I was a little surprised no one mentioned taking over maintainer-ship of at least this file as part of our previous discussions.